### PR TITLE
chore: use library template style for info pages

### PR DESCRIPTION
**Pull request by mhaack**
Related #468

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.live/us/en/info/what-is-luminescence
- After: https://feat-468-info-template--danaher-ls-aem--hlxsites.hlx.live/us/en/info/what-is-luminescence

### CODE REVIEW

**Review by davenichols-DHLS — APPROVED** [review]
Hi @mhaack , this is great, thanks.  However, the text colour and some styling is slightly different?  For example the text should be dark grey like on the Blogs and Library pages.

**Comment by mhaack** [comment]
> Hi @mhaack , this is great, thanks. However, the text colour and some styling is slightly different? For example the text should be dark grey like on the Blogs and Library pages.

Looking into it

File: styles/tailwind.css [hunk 1]
```diff
@@ -157,55 +147,6 @@
...
-
-  .btn-trending-brand {
-    @apply text-white bg-danaherpurple-500 hover:bg-danaherpurple-800 border-danaherpurple-500;
-  }
-
-  .btn-outline-brand {
-    @apply text-danaherorange-500 bg-white border-2 border-danaherorange-500 hover:text-white hover:bg-danaherorange-500;
-  }
-
-  .btn-outline-trending-brand {
```
[the remark this answers is not in the document]
Comment (mhaack) [reply styles/tailwind.css:180]: It is still there https://github.com/hlxsites/danaher-ls-aem/pull/486/files#diff-68dadc54d8175dc515f314c61980dcf2173c4699233d034eab401dbe8b4a0677R158

[the remark this answers is not in the document]
Comment (duynguyen) [reply styles/tailwind.css:180]: got confused by the UI. sorry about the noise.

**Review by duynguyen — APPROVED** [review]

**Pull request merged**